### PR TITLE
#3788 - Gateway Assessment Data Update - PT & FT

### PR DESCRIPTION
**Pull request by andrewsignori-aot**
- Updated values for part-time and full-time.
- No E2E adjustments were needed.

## Part-time

Please see below the offering book and supplies (a.k.a. program-related costs) adjusted to the assessment's value.

image

image

### Sample Assessment-related Data
```json
{
  "weeks": 31,
  "tuitionCost": 5500,
  "childcareCost": 5376.08,
  "mandatoryFees": 350,
  "finalAwardTotal": 13767.08,
  "totalFamilyIncome": 24000,
  "transportationCost": 403,
  "totalAssessmentNeed": 13767.08,
  "booksAndSuppliesCost": 1798,
  "totalAcademicExpenses": 8391,
  "miscellaneousAllowance": 340,
  "exceptionalEducationCost": 600,
  "finalFederalAwardNetCSGDAmount": 1736,
  "finalFederalAwardNetCSGPAmount": 0,
  "finalFederalAwardNetCSLPAmount": 9511.08,
  "finalFederalAwardNetCSPTAmount": 2520,
  "finalProvincialAwardNetBCAGAmount": 0,
  "finalProvincialAwardNetSBSDAmount": 0
}
```

## Full-time

Please see below some values from the full-time NOA generated. The NOA changes for full-time are not critical at the moment and the NOA must be adjusted. For instance, federal and provincial awards are just duplicated right now and they display the same values.

image

### CODE REVIEW

File: sources/packages/backend/workflow/src/workflow-definitions/assessment-gateway.bpmn [hunk 1]
```diff
@@ -1,5 +1,5 @@
...
+<bpmn:definitions xmlns:bpmn="http://www.omg.org/spec/BPMN/20100524/MODEL" xmlns:bpmndi="http://www.omg.org/spec/BPMN/20100524/DI" xmlns:dc="http://www.omg.org/spec/DD/20100524/DC" xmlns:zeebe="http://camunda.org/schema/zeebe/1.0" xmlns:di="http://www.omg.org/spec/DD/20100524/DI" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:modeler="http://camunda.org/schema/modeler/1.0" xmlns:camunda="http://camunda.org/schema/1.0/bpmn" id="Definitions_1" targetNamespace="http://bpmn.io/schema/bpmn" exporter="Camunda Modeler" exporterVersion="5.28.0" modeler:executionPlatform="Camunda Cloud" modeler:executionPlatformVersion="8.6.0" camunda:diagramRelationId="f874f098-96b6-491d-81ec-ecf7a0d8c67a">
```
Comment (dheepak-aot) [inline sources/packages/backend/workflow/src/workflow-definitions/assessment-gateway.bpmn:2]: 👍

File: sources/packages/backend/workflow/src/workflow-definitions/assessment-gateway.bpmn [hunk 2]
```diff
@@ -306,7 +306,7 @@
...
+          <zeebe:input source="={&#10;  weeks: offeringWeeks,&#10;  tuitionCost: calculatedDataActualTuitionCosts,&#10;  booksAndSuppliesCost: calculatedDataProgramRelatedCosts,&#10;  mandatoryFees: calculatedDataMandatoryFees,&#10;  exceptionalEducationCost: offeringExceptionalExpenses,&#10;  childcareCost: calculatedDataTotalChildCareCost,&#10;  totalFamilyIncome: calculatedDataTotalFamilyIncome,&#10;  transportationCost: calculatedDataTotalTransportationAllowance,&#10;  miscellaneousAllowance: calculatedDataMiscellaneousAllowance,&#10;  totalAssessmentNeed: calculatedDataTotalAssessedNeed,&#10;  totalAcademicExpenses: calculatedDataTotalAcademicExpenses,&#10;  finalFederalAwardNetCSGPAmount: finalFederalAwardNetCSGPAmount,&#10;  finalFederalAwardNetCSPTAmount: finalFederalAwardNetCSPTAmount,&#10;  finalFederalAwardNetCSGDAmount: finalFederalAwardNetCSGDAmount,&#10;  finalProvincialAwardNetBCAGAmount: finalProvincialAwardNetBCAGAmount,&#10;  finalProvincialAwardNetSBSDAmount: finalProvincialAwardNetSBSDAmount,&#10;  finalFederalAwardNetCSLPAmount: finalFederalAwardNetCSLPAmount,&#10;  finalAwardTotal: finalAwardTotal&#10;}" target="assessmentData" />
```
Comment (dheepak-aot) [inline sources/packages/backend/workflow/src/workflow-definitions/assessment-gateway.bpmn:309]: Do we require the exceptional expense for the PT assessment data at all? 
image

**Review by dheepak-aot — APPROVED** [review]
Good job! Just a minor question. No blockers.

File: sources/packages/backend/workflow/src/workflow-definitions/assessment-gateway.bpmn (hunk 2 above)
Comment (andrewsignori-aot) [reply sources/packages/backend/workflow/src/workflow-definitions/assessment-gateway.bpmn:309]: We do not require it in the NOA but we do for some data inventory reports (that may be removed in the future).
I would not remove it right now.

**Review by guru-aot — APPROVED** [review]
LGTM, Nice work @andrewsignori-aot

**Pull request merged**